core/tests/webdriverio_desktop/extensions.js

```diff
@@ -72,6 +72,10 @@ describe('rich-text components', function() {
       }]);
       // Test highlighting on Skillreview component.
       await richTextEditor.appendPlainText('highlight');
+      // Pre check to ensure that no text is currently highlighted.
+      expect(await browser.execute(() => {
+        return window.getSelection();
+      })).toBe(null);
```

Comment:
We like to keep direct interactions with the page in `webdriverio_utils/`, so can you move this logic there?

Reply:
Done